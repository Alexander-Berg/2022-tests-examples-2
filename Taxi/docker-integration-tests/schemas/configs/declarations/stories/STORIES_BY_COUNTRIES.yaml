description: 'Правила отображения сториз по странам. Список контекстов к которым применяются
    правила. Если контекста в списке нет, правила к нему не применяются'
default:
    safety_center:
        __all__: []
tags: []
schema:
    type: object
    additionalProperties:
        $ref: '#/definitions/Context'
    definitions:
        Country:
            type: object
            properties:
                include:
                    description: 'id сториз которые будут дополнительно отображаться
                        в этой стране'
                    type: array
                    items:
                        type: string
                exclude:
                    description: 'id сториз из общего списка которые не будут отображаться
                        в этой стране'
                    type: array
                    items:
                        type: string
            additionalProperties: false
        Context:
            description: 'Контекст к которому применяется правило'
            type: object
            properties:
                __all__:
                    description: 'Сториз с этими id будут отображаться во всех странах'
                    type: array
                    items:
                        type: string
            additionalProperties:
                $ref: '#/definitions/Country'
            required:
              - __all__
