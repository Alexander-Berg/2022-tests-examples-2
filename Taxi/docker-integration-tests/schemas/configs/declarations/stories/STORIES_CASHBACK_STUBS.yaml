description: 'Временные сториз для кэшбека'
default:
    stories: []
tags: []
schema:
    type: object
    additionalProperties: false
    properties:
        stories:
            type: array
            items:
                $ref: '#/definitions/StoryObject'
    required:
      - stories
    definitions:
        StoryObject:
            type: object
            additionalProperties: false
            properties:
                id:
                    type: string
                    description: 'Mongo story id'
                created:
                    type: string
                updated:
                    type: string
                name:
                    type: string
                button_title_key:
                    type: string
                    description: 'Tanker key for button text translation'
                languages:
                    type: array
                    items:
                        $ref: '#/definitions/StoriesLocalization'
                active:
                    type: boolean
                    description: 'Is marked active (unused)'
            required:
              - id
              - name
              - button_title_key
              - languages

        StoriesLocalization:
            type: object
            additionalProperties: false
            properties:
                button_link:
                    type: string
                    description: 'button link'
                    example: 'https://taxi.yandex.ru/action/b2b-friend-referral/?utm_source=app&utm_campaign=stories_app&utm_medium=referal'
                media:
                    type: array
                    items:
                        $ref: '#/definitions/MediaObject'
                    description: 'array of stories media'
                locale:
                    type: string
                    description: 'stories name'
                    example: '080419_3years_b2b'
                teaser_image:
                    type: string
                    description: 'teaser image link'
                    example: 'https://promo-stories.s3.yandex.net/080419_3years_b2b/ru/9e4f4f79f99667f07100939e1d6fddc1770bd116.png'
            required:
              - locale
              - teaser_image
              - media

        MediaObject:
            type: object
            additionalProperties: false
            properties:
                content:
                    type: string
                    description: 'link to media'
                    example: 'https://promo-stories.s3.yandex.net/080419_3years_b2b/ru/bfbfcf8ba55b554ce64470e8c4ac7645db26a9d7.mp4'
                show_button:
                    type: boolean
                    description: 'True if we need to show StoryObject.button while
                        playing this media'
                type:
                    type: string
                    description: 'media content type'
                    example: 'video'
