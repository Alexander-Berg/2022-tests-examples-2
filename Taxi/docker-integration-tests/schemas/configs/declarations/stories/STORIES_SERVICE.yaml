description: 'Stories service settings'
default:
    default_locale: 'ru'

tags: []
schema:
    type: object
    additionalProperties: false
    properties:
        default_locale:
            type: string
            description: 'Language to use when accept-language is not found in localizations'
    required:
      - default_locale
