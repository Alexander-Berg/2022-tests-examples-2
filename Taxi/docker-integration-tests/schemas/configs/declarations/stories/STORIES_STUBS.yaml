description: 'Временные сториз для центра безопасности'
default:
    stories: []
tags: []
schema:
    type: object
    additionalProperties: false
    properties:
        stories:
            type: array
            items:
                $ref: '#/definitions/StoryObject'
    definitions:
        StoryObject:
            type: object
            additionalProperties: false
            properties:
                button_link:
                    type: string
                    description: 'button link'
                    example: 'https://taxi.yandex.ru/action/b2b-friend-referral/?utm_source=app&utm_campaign=stories_app&utm_medium=referal'
                button_title:
                    type: string
                    description: 'button title'
                    example: "Рекомендовать"
                media:
                    type: array
                    items:
                        $ref: '#/definitions/MediaObject'
                    description: 'array of stories media'
                name:
                    type: string
                    description: 'stories name'
                    example: '080419_3years_b2b'
                teaser_image:
                    type: string
                    description: 'teaser image link'
                    example: 'https://promo-stories.s3.yandex.net/080419_3years_b2b/ru/9e4f4f79f99667f07100939e1d6fddc1770bd116.png'
        MediaObject:
            type: object
            additionalProperties: false
            properties:
                content:
                    type: string
                    description: 'link to media'
                    example: 'https://promo-stories.s3.yandex.net/080419_3years_b2b/ru/bfbfcf8ba55b554ce64470e8c4ac7645db26a9d7.mp4'
                show_button:
                    type: boolean
                    description: 'True if we need to show StoryObject.button while
                        playing this media'
                type:
                    type: string
                    description: 'media content type'
                    example: 'video'

