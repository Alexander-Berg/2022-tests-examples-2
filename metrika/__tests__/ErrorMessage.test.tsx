import * as React from 'react';

import { makeScreenshot } from '@metrika/ui/utils/jest/makeScreenshot';

import { Main } from '../ErrorMessage.stories';

describe('ErrorMessage', () => {
    it('screenshooter', async () => {
        await makeScreenshot(<Main />);
    });
});
