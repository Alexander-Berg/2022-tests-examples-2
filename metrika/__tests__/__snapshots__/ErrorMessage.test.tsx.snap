// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage screenshooter 1`] = `
"<style data-styled=\\"true\\" data-styled-version=\\"5.2.0\\">.ivKWIK{font-size:16px;line-height:20px;-webkit-letter-spacing:0.06px;-moz-letter-spacing:0.06px;-ms-letter-spacing:0.06px;letter-spacing:0.06px;color:#939cb0;}/*!sc*/
data-styled.g1[id=\\"sc-bdnylx\\"]{content:\\"ivKWIK,\\"}/*!sc*/
</style>
<div class=\\"sc-bdnylx ivKWIK\\">Whoops, looks like error!</div>"
`;
